readme.md

```diff
@@ -45,6 +45,12 @@ Type: `number`
 
 The maximum number of items before evicting the least recently used items.
 
+#### onEviction
+*Optional*\
+Type: `(key, value) => void`
+
+Method to call when an item is about to be evicted from the cache. Useful when a cleanup operation is necessary.
```

Comment:
Can you describe what kind of cleanup operation you would use it for? It's not always clear to users what they would use an option for. It helps to provide some example use-cases.